tests/e2e/utils/customer.js

```diff
@@ -42,8 +42,8 @@ export async function singleProductAddToCart( page, productID ) {
  */
 export async function relatedProductAddToCart( page ) {
 	const addToCart = ( await page.locator( '.related.products' ).isVisible() )
-		? '.related.products .add_to_cart_button'
-		: '.wp-block-woocommerce-related-products .add_to_cart_button';
+		? '.related.products .add_to_cart_button.product_type_simple'
+		: '.wp-block-woocommerce-related-products .add_to_cart_button.product_type_simple';
```

Comment:
Since this function is made to work only for simple products, I think we should add a comment stating that only will work for simple products.